actix-web-codegen/src/route.rs

```diff
@@ -166,6 +199,13 @@ impl Route {
 
         let args = Args::new(args)?;
 
+        if guard == GuardType::Multi && args.methods.is_empty() {
+            return Err(syn::Error::new(
+                Span::call_site(),
+                "The #[route(..)] macro requires at least one `method` attribute!",
```

Comment:
Let's keep ourselves calm 🙃 

```suggestion
                "The #[route(..)] macro requires at least one `method` attribute",
```